src/mhitm.c

```diff
@@ -313,7 +311,7 @@ mattackm(register struct monst *magr, register struct monst *mdef)
     tmp = find_mac(mdef) + magr->m_lev;
     if (mdef->mconf || !mdef->mcanmove || mdef->msleeping) {
         tmp += 4;
-        wakeup(mdef, FALSE);
+        wakeup(mdef, FALSE, FALSE);
```

Comment:
Does it make sense to keep them cloaked here? Won't it look like magr attacking a random piece of furniture or object?